action.yml

```diff
@@ -10,6 +10,10 @@ inputs:
   parameters:
     description: "Deployment parameters eg. Instance name"
     required: true
+  api_endpoint:
+    description: "API URL"
+    required: false
+    default: "https://api.zrutech.ca/builder/instance/api/"
```

Comment:
can this just be `https://api.zrutech.ca` and the actual URI be part of the index.js?